addon/components/visualizer-course-objectives.hbs

```diff
@@ -1,48 +1,55 @@
-{{! template-lint-disable attribute-indentation }}
-{{#if (or true @isIcon (gt (get (await this.objectiveWithMinutes) "length") 0))}}
-  <SimpleChart
-    @name="donut"
-    @isIcon={{@isIcon}}
-    @data={{await this.objectiveWithMinutes}}
-    @hover={{perform this.donutHover}}
-    @leave={{perform this.donutHover}} as |chart|
-  >
-    {{#if this.tooltipContent}}
-      <chart.tooltip @title={{this.tooltipTitle}}>
-        {{this.tooltipContent}}
-      </chart.tooltip>
+<div
+  class="visualizer-course-objectives {{if (not @isIcon) "not-icon"}}"
+  {{did-insert (perform this.load) @course}}
+  {{did-update (perform this.load) @course}}
+>
+  {{#unless this.load.isRunning}}
+    {{#if (or true @isIcon this.objectiveWithMinutes.length)}}
```

Comment:
I'm confused by this repeating line. Seems like `(or true` will always be true. Maybe a testing artifact?

Reply:
yea i was wondering about those too. will see how removing this will affect the whole thing.

Reply:
yanked `true` from those conditional checks without repercussions.